### ISSUE TITLE
exonerate download link is broken

### ISSUE THREAD

**Issue by mossmatters** [Homebrew/homebrew-science#2636]
```
$ brew reinstall exonerate

==> Reinstalling exonerate

==> Downloading http://www.ebi.ac.uk/~guy/exonerate/exonerate-2.2.0.tar.gz

curl: (22) The requested URL returned error: 404 Not Found

Error: Failed to download resource "exonerate"
Download failed: http://www.ebi.ac.uk/~guy/exonerate/exonerate-2.2.0.tar.gz
```

The website for exonerate is no longer supported, as the base website is apparently down for good: http://www.ebi.ac.uk/~guy/exonerate/exonerate-2.2.0.tar.gz

There is a github repository, although only for the newer 2.4.0 version: https://github.com/nathanweeks/exonerate

I was able to compile from the github source on MacOS 10.10.4 using: 

```
./configure
automake --add-missing 
make
make install
```

**Comment by dunn** [comment]
If that GitHub repo is the new official upstream, then we can just update the formula to 2.4.0.  Feel like opening a pull request?

**Comment by tdsmith** [comment]
It's a fork, but the upstream's gone missing.

**Comment by dpo** [comment]
New repo hasn't had a commit since January, but an issue was open there today. Should we consider it as the new upstream?

**Comment by andrewjpage** [comment]
The people responsible for the upstream bit are aware of the issue, but are on holidays at the moment so cant fix it for a while.  For those of you who dont live in Europe, August is when most people take a few weeks off.

**Comment by andrewyatz** [comment]
Fix just submitted to PR #2686
